src/azul/service/responseobjects/buffer.py

```diff
@@ -0,0 +1,38 @@
+from logging import getLogger
+from io import BytesIO
+from typing import Callable
+
+logger = getLogger(__name__)
+
+
+class BytesBuffer(BytesIO):
```

Comment:
The name is very generic so as a reader I don't get the essential purpose. Document the purpose in one or two sentences.